modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/MakeAlphaFoldDBProteinFeatures.pm

```diff
@@ -0,0 +1,237 @@
+=head1 LICENSE
+
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+=head1 CONTACT
+
+  Please email comments or questions to the public Ensembl
+  developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+  Questions may also be sent to the Ensembl help desk at
+  <http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+=head1 NAME
+
+Bio::EnsEMBL::Analysis::Runnable::MakeAlphaFoldDBProteinFeatures -
+
+=head1 SYNOPSIS
+
+
+=head1 DESCRIPTION
+
+This module makes protein features based on the AFDB-UniProt mappings found in the EMBL-EBI AFDB SIFTS data and the UniProt-ENSP mappings found in the GIFTS database in order to make the link between AFDB and ENSP having a AFDB entry as a protein feature for a given ENSP protein.
+
+=head1 APPENDIX
+
+The rest of the documentation details each of the object methods.
+Internal methods are usually preceded with a _
+
+=cut
+
+package Bio::EnsEMBL::Analysis::Runnable::MakeAlphaFoldDBProteinFeatures;
```

Comment:
Package name to change to new path Bio/EnsEMBL/Production/Pipeline/AlphaFold/MakeAlphaFoldDBProteinFeatures